var/spack/repos/builtin/packages/spdlog/package.py

```diff
@@ -49,15 +49,22 @@ class Spdlog(CMakePackage):
     depends_on('cmake@3.2:', when='@:1.7.0', type='build')
     depends_on('cmake@3.10:', when='@1.8.0:', type='build')
 
+    variant('external_fmt', default=False,
```

Comment:
Default should probably be true for spack

Reply:
I'd say this shouldn't be a variant at all: packages in spack should not have vendored dependencies. The only exceptions would be allowed for unusual cases where the dependency's symbols are mangled so that they can't conflict with another instance in the build.

Reply:
Ok.  spdlog doesn't do any mangling and I ran into issues when using a package that required both fmt and spdlog.  I'll double check to see if there are restrictions on the fmt versions for spdlog before fixing the PR

Reply:
Some digging shows that only as of version 1.4.1 did `SPDLOG_FMT_EXTERNAL` do the right thing and not install its bundled fmt: https://github.com/gabime/spdlog/pull/1241. Should I remove the versions prior in the updated PR?

Reply:
The updated PR removes the options to use spdlog's internal fmt version and restricts the external fmt version based on spdlog's release notes.

Reply:
Probably a variant would have been good, seems that some packages seem to pull in that bundled fmt :exploding_head: 
https://github.com/spack/spack/pull/30051#issuecomment-1102500778